cleverpush/src/main/java/com/cleverpush/database/TableBannerTrackEventDao.java

```diff
@@ -19,20 +19,20 @@ public interface TableBannerTrackEventDao {
   @Update
   void update(TableBannerTrackEvent bannerTrackEvent);
 
-  @Query("SELECT * FROM TableBannerTrackEvent WHERE bannerId = :bannerId AND trackEventId = :trackEventId "
+  @Query("SELECT * FROM cleverpush_table_banner_track_event WHERE bannerId = :bannerId AND trackEventId = :trackEventId "
```

Comment:
Do we need `bannerId` in the table? Shouldn't the list of tracked events + properties be enough?

Reply:
Yes, we need a banner ID for the count. I have 2 banners with the same event ID same property, and a different count than we need banner ID
for 1st banner, the past 3 days for the second banner past 5 days.

<img width="1082" alt="Screenshot 2023-11-28 at 8 00 34 PM" src="https://github.com/cleverpush/cleverpush-android-sdk/assets/129381063/32d0ffbd-03b9-4b1b-b4bb-a1e4bfb5ba08">

<img width="1107" alt="Screenshot 2023-11-28 at 8 01 54 PM" src="https://github.com/cleverpush/cleverpush-android-sdk/assets/129381063/acd69f4a-26a1-4e48-a359-003ebe1b4bab">